src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -111,6 +111,22 @@ public DynamoView(DynamoViewModel dynamoViewModel)
             SizeChanged += DynamoView_SizeChanged;
             LocationChanged += DynamoView_LocationChanged;
 
+            if (LibraryCollapsed)
+            {
+                var imageUri = new Uri(@"pack://application:,,,/DynamoCoreWpf;component/UI/Images/expand_normal.png");
+                BitmapImage icon = new BitmapImage(imageUri);
+                LibrarySidebarIcon.Source = icon;
+                LibrarySidebarText.Visibility = Visibility.Visible;
```

Comment:
not a big deal here, but the concept of ViewModel / Model pattern could be followed here. 
For ex: LibraryCollapsed is in xaml.cs and it updates LibraryCollapsed in ViewModel. Based on the value, Source, Visibility gets updated.

Reply:
Thanks @ramramps, that's a good point.  I am going to do a little additional cleanup since this will be demo'd next sprint.  I will also address the icon swap when user drags outside of the expected threshold.